tests/integration/REST_API/Controllers/SettingsTest.php

```diff
@@ -38,6 +38,82 @@ protected function _after() {
 	/** Tests *********************************************************************************************************/
 
 
+	/** @see Settings::get_item() */
+	public function test_get_item() {
+
+		$settings   = $this->get_settings_instance();
+		$controller = new Settings( $settings );
+
+		$setting_id = 'test';
+
+		$settings->register_setting( $setting_id, Setting::TYPE_STRING, [
+			'name'        => 'Test Setting',
+			'description' => 'A simple setting',
+			'options'     => [ 'a', 'b', 'c' ],
+			'default'     => 'c',
+		] );
+
+		$settings->register_control( $setting_id, Control::TYPE_SELECT, [
+			'name'        => 'Select field',
+			'description' => 'A regular select input field',
+			'options'     => [
+				'a' => 'A',
+				'b' => 'B',
+				'c' => 'C'
+			],
+		] );
+
+		$setting = $settings->get_setting( $setting_id );
+		$control = $setting->get_control();
+
+		$setting->set_value( 'a' );
+
+		$request = new WP_REST_Request( 'GET', "/wc/v3/{$settings->get_id()}/settings" );
+		$request->set_url_params( [ 'id' => $setting_id ] );
```

Comment:
I am not 100% sure this is the correct way to send the param.

Reply:
I think that's how it's done internally.